harvard-university-of-leeds.csl

```diff
@@ -32,7 +36,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="text" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all"/>
+      <name and="text" delimiter-precedes-last="never" et-al-min="0" initialize-with="." name-as-sort-order="all"/>
```

Comment:
@adam3smith - I've just been clarifying this.
The in-text citation should have 'et al' if there are 3 or more authors, and in which case only the first author should be listed.

The change should actually be to the bibliography section.
The guidance is
> **All author names should be included in the reference.**
> Where a source has a very long list of authors, eg in the case of some scientific articles, 
> you might wish to consult your tutor on whether to use "et al" in place of some author names.

Trying to code 'ask your tutor' into a CSL is a bit like trying to knit fog.
Our Endnote style has a limitless author list for the bibliography, as much as I think it's silly, I think the same should be applied here!